### PR TITLE
Create zh_CN.lang

### PR DESCRIPTION
**Pull request by SihenZhang**
By the way, there are some problems that made me confused.

1. Line 28 : ```industrialwires.chat.stackSize=Linking is only possible with a stack of size 1```
I have tried so many times in varieties of ways, but this hint never appeared. Could you tell me when it will appear because it will make a difference to my translation.

2. Line 36: ```ie.manual.entry.industrialWires.wires.subtext=No complex impedance!```
I think it means that with industrial wires, there will be no impedance anymore. But in practice, impedance is also existent. So what's the meaning of this sentence?

3. Line 43: ```ie.manual.entry.industrialWires.mechConv0=Both the IC2 ... To use the "Converter: Rotational To Kinetic" attach a source of IE rotational energy```
Maybe a comma was left out in front of "attach"  :)

4. Line 44: ```ie.manual.entry.industrialWires.mechConv1=like a waterwheel ... (Rotational and kinetic energy have to be swapped)<br> ...```
What's the meaning of "have to"? Or what's the meaning of the sentence in brackets? Does "have to" mean "must"?

5. All of the Mechanical Converters' textures are missing when placed. 
Image of Missing Textures 
Hope to fix.

Looking forward to your reply and do NOT merge because I need to finish my translation.

### CODE REVIEW

**Comment by malte0811** [comment]
First of all, thank you for translating the localization! Since I don't know any Chinese I would prefer to have someone else check the translation once you are done, probably the person who translated the IE localization (3TUSK).
Regarding your problems:
1. The message should be shown in the chat when you try to link connectors using a stack of more than one coil (e.g. 10 coils of 20 blocks of tin wire each (Coils with the same amount of wire can stack)).
2. That is a reference to [electrical impedance](https://en.wikipedia.org/wiki/Electrical_impedance). It is a bit like electrical resistance (which is "real" impedance), but accounts for capacitance and inductance as well (Those are "complex" impedance). Real-world power lines have capacitance and inductance, but the in-game wires only have resistance (well, sort of... At least they lose energy), so they don't have complex impedance.
3. Yes, I wasn't sure whether one was needed, but it certainly wouldn't hurt to add that comma.
4. That means that to use the "Converter: Kinetic To Rotational", you set it up like the "Converter: Rotational To Kinetic" except that you swap rotational and kinetic energy.
5. It is working for me, in a "normal" (non-dev) environment. Could you open an issue about that and include a full log?

File: src/main/resources/assets/industrialwires/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+item.industrialwires.ic2wireCoil.gold.name=金质线圈
+item.industrialwires.ic2wireCoil.hv.name=工业2高压线圈
+item.industrialwires.ic2wireCoil.glass.name=玻璃纤维线圈
+
+
+industrialwires.desc.wireLength=线缆长度: %1s 格
+industrialwires.desc.recipe=请查看工程师手册以获取配方详情
+
+industrialwires.chat.tooLong=该线圈的长度不足以完成此次连接
+industrialwires.chat.stackSize=Linking is only possible with a stack of size 1
```
Comment (3TUSK) [inline src/main/resources/assets/industrialwires/lang/zh_CN.lang]: 连接时只能使用一个线圈

File: src/main/resources/assets/industrialwires/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+ 
+itemGroup.industrialwires=工业线缆
+
+ie.manual.category.industrialWires.name=工业线缆
+
+ie.manual.entry.industrialWires.wires.name=工业线缆
+ie.manual.entry.industrialWires.wires.subtext=No complex impedance!
```
Comment (3TUSK) [inline src/main/resources/assets/industrialwires/lang/zh_CN.lang]: 没有复杂的[阻抗](https://zh.wikipedia.org/wiki/%E9%98%BB%E6%8A%97)概念！

File: src/main/resources/assets/industrialwires/lang/zh_CN.lang [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ie.manual.entry.industrialWires.wires0=工业线缆mod的线缆可以像工业2的导线一样传输能量。<br>其使用方法与使用ImmersiveEngineering的线缆完全一样。每种线缆所能传输的EU与其对应的导线相同，所以
+ie.manual.entry.industrialWires.wires1=将接线器接在一个会损坏导线的电源上同样会损坏接线器。<br>用以充当工业2导线的线圈与ImmersiveEngineering的线圈不同，距离越长，所使用的线缆也会越长：线圈是通过在合成格中放置任意组合的未绝缘的工业2导线和相应的线圈来合成的。下一页展示了一些有效的配方及其合成产物的例子。
+ie.manual.entry.industrialWires.wires2=未绝缘的锡质导线可以由未绝缘的铜质导线、金质导线、高压导线或玻璃纤维导线替代，以制造其他线圈。
+
+ie.manual.entry.industrialWires.mechConv.name=机械能转换器
+ie.manual.entry.industrialWires.mechConv.subtext=我为此发明了旋转能!
+ie.manual.entry.industrialWires.mechConv0=工业2和IE都有产生某种动能的机器。工业线缆mod的新型转换器允许您在这两种形式的能量之间转换！<br>要使用“转换器：旋转能-动能”，需要将其齿轮面与一个IE的旋转能源
+ie.manual.entry.industrialWires.mechConv1=如水车或旋转能马达（见第3页）相连，其对面与一个工业2消耗动能的机器相连。“转换器：动能-旋转能”的使用方法与“转换器：旋转能-动能”相似（Rotational and kinetic energy have to be swapped）。<br>不幸的是，转换过程中会
```
Comment (3TUSK) [inline src/main/resources/assets/industrialwires/lang/zh_CN.lang]: ...“转换器：IC动能-IE旋转能”的使用方法与“转换器：IE旋转能-IC动能”相似，*只是转换方向变成了IC动能到IE旋转能*... （No parentheses）

**Comment by 3TUSK** [comment]
I am summoned.

Beside, for 4.:
>Rotational and kinetic energy have to be swapped   

I would say it in this way: *...except the direction of conversion was inverse...*, without parentheses.  

> Guess I may start on making documentation of IndustrialWires on ftb.gamepedia.com ...

**Comment by malte0811** [comment]
Thank you for reviewing this @3TUSK.
I will merge this now, I have to remember to add a "credits"-file later.

**Pull request merged**